### PR TITLE
add the controller for meetings only have post or talk

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Give the ability for single meeting to only have poster or talk. Closes https://github.com/CenterForOpenScience/osf.io/issues/1365.

### CODE REVIEW

File: website/templates/public/pages/meeting_body.mako [hunk 1]
```diff
@@ -1,4 +1,6 @@
-<h2 style="padding-bottom: 30px;">${ meeting['name'] } Posters & Talks</h2>
+<h2 style="padding-bottom: 30px;">${ meeting['name'] }
+    <a>Posters & Talks if meeting['poster'] and meeting['talk'] else Posters if meeting['poster'] else Talks</a>
```
Comment (sloria) [inline website/templates/public/pages/meeting_body.mako]: This doesn't appear correct; there are no `${}` delimiters.

Please make sure to test this locally.

File: website/templates/public/pages/meeting_body.mako [hunk 2]
```diff
@@ -7,21 +9,26 @@
 
 % if meeting['active']:
     <div>
-        <a id="addLink" onclick="" href="#">Add your poster or talk</a>
+        <a id="addLink" onclick="" href="#">Add your posters or talks if meeting['poster'] and meeting['talk'] else add your posters if meeting['poster'] else add your talks</a>
```
Comment (sloria) [inline website/templates/public/pages/meeting_body.mako]: Same here; is this mako correct?

File: website/templates/public/pages/meeting_body.mako [hunk 3]
```diff
@@ -7,21 +9,26 @@
...
+
         % if meeting['info_url']:
           | <a href="${ meeting['info_url'] }" target="_blank">Conference homepage <i class="fa fa-sm fa fa-external-link"></i></a>
         % endif
     </div>
 
     <div style="display: none" id="submit">
-        <h3>Add your poster or talk</h3>
+        <h3>Add your posters or talks if meeting['poster'] and meeting['talk'] else add your posters if meeting['poster'] else add your talks</h3>
```
Comment (sloria) [inline website/templates/public/pages/meeting_body.mako]: See previous comments.

**Comment by sloria** [comment]
Please double-check your syntax here and test locally. When that's done, I'll do next pass.

File: website/templates/public/pages/meeting_body.mako (hunk 1 above)
Comment (chennan47) [inline website/templates/public/pages/meeting_body.mako]: addressed in 69b61c5

**Comment by chennan47** [comment]
@sloria rdy for review

**Comment by HarryRybacki** [comment]
[#OSF-4211](https://openscience.atlassian.net/browse/OSF-4211)

File: website/templates/public/pages/meeting_body.mako [hunk 4]
```diff
@@ -1,4 +1,6 @@
-<h2 style="padding-bottom: 30px;">${ meeting['name'] } Posters & Talks</h2>
+<h2 style="padding-bottom: 30px;">${ meeting['name'] }
+    <a>${'Posters & Talks' if meeting['poster'] and meeting['talk'] else 'Posters' if meeting['poster'] else 'Talks'}</a>
```
Comment (abought) [inline website/templates/public/pages/meeting_body.mako]: Why is this a link? No destination is specified, and clicking it doesn't seem to do anything on the live page.

File: website/templates/public/pages/meeting_body.mako [hunk 5]
```diff
@@ -7,21 +9,26 @@
...
         % if meeting['info_url']:
           | <a href="${ meeting['info_url'] }" target="_blank">Conference homepage <i class="fa fa-sm fa fa-external-link"></i></a>
         % endif
     </div>
 
     <div style="display: none" id="submit">
-        <h3>Add your poster or talk</h3>
+        <h3>${'Add your posters or talks' if meeting['poster'] and meeting['talk'] else 'Add your posters' if meeting['poster'] else 'Add your talks'}</h3>
         <p>
             Send an email to one of the following addresses from the email
```
Comment (abought) [inline website/templates/public/pages/meeting_body.mako]: It says "one of the following addresses", even for conferences where there is now only one item in the list.

**Comment by abought** [comment]
Pass finished.

Two small comments, but otherwise this looks good. 

Please always run your code and examine the page before submitting something to code review. (the script workarounds we discussed yesterday should work)

File: website/templates/public/pages/meeting_body.mako (hunk 4 above)
Comment (chennan47) [inline website/templates/public/pages/meeting_body.mako]: addressed in 43198f8

File: website/templates/public/pages/meeting_body.mako (hunk 5 above)
Comment (chennan47) [inline website/templates/public/pages/meeting_body.mako]: addressed in 43198f8

**Comment by abought** [comment]
Changes look good to me. :shipit:

**Comment by sloria** [comment]
In order for us to test this on staging, the `dev_populate_conferences.py` script will also need to be updated. @chennan47 Can you look into this?

**Comment by chennan47** [comment]
@sloria update done. Rdy for another pass.

**Pull request merged**